examples/gno.land/p/demo/accesscontrol/accesscontrol_test.gno

```diff
@@ -0,0 +1,227 @@
+package accesscontrol
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+// TestAccessControl verifies the access control functionality.
+func TestAccessControl(t *testing.T) {
+	admin := testutils.TestAddress("admin")
+	user1 := testutils.TestAddress("user1")
+	user2 := testutils.TestAddress("user2")
+
+	// Create new RoleData
+	roleData := NewRole("admin", admin)
+
+	// Check initial admin role
+	if roleData.AdminRole != admin {
+		t.Fatalf("expected admin role to be %s, got %s", admin.String(), roleData.AdminRole.String())
+	}
```

Comment:
```suggestion
	uassert.NotEqual(t, roleData.AdminRole, admin)
```

Reply:
I change for `!uassert.Equal()` because `uassert.NotEqual()` is not merged 
[e94a0b4](https://github.com/gnolang/gno/pull/2307/commits/e94a0b48301ab2e57aae011f3909f1fe7c2baaca)